src/mymonero/HostedMoneroAPIClient/HostedMoneroAPIClient_Base.js

```diff
@@ -29,8 +29,8 @@
 "use strict"
 //
 //
-const JSBigInt = require('../cryptonote_utils/biginteger.js').BigInteger // important: grab defined export
-const monero_config = require('../monero_utils/monero_config.js')
+const JSBigInt = require('@mymonero/mymonero-bigint').BigInteger // important: grab defined export
+const monero_config = require('@mymonero/mymonero-money-format/monero_config')
 const net_service_utils = require('../hostAPI/net_service_utils.js')
```

Comment:
Why not replace this with "https://www.npmjs.com/package/@mymonero/mymonero-net-service-utils" ?

Reply:
Because our forked copy has been modified to be `async`, where theirs expects the C++ methods to be sync (because WASM).